beetsplug/discogs.py

```diff
@@ -434,9 +434,12 @@ def get_album_info(self, result):
 
         # Extract information for the optional AlbumInfo fields that are
         # contained on nested discogs fields.
-        media, albumtype = self.get_media_and_albumtype(
-            result.data.get("formats")
-        )
+        albumtypes = albumtype = media = label = catalogno = labelid = None
```

Comment:
You'd want to set `albumtypes = []` for clarity here.

Actually `discogs` plugin has been updated heavily on my end, including calculation of the `albumtypes` field. Let me try to dig this out

Reply:
Maybe we should leave this out for another PR, I'm thinking?

Reply:
Yeah exactly, had the same thought. I'm debugging around here already like crazy....but so far I think this PR is ready to merge.

Please reset or revert the branch to what it was before my commit and I think we should merge it now and note in an issue that there might something to fix with discogs plug.

Reply:
Cool, will do this now!

Reply:
Just need an approval and we're good to go for now.

Reply:
One thing: Were would be documented that users need to reimport? Maybe a hint in the changelog? Something like: Read PR description how to fix?

Reply:
Good point, I'll give this a mention in the changelog!

Reply:
done 👍🏼